src/include/ipc/dai-amd.h

```diff
@@ -23,5 +23,6 @@ struct sof_ipc_dai_acp_params {
 	uint32_t reserved0;
 	uint32_t fsync_rate;
 	uint32_t tdm_slots;
+	uint32_t tdm_mode;
```

Comment:
Is there a kernel PR for this part ?

Reply:
Yes ,its already merged to sof-linux.

https://github.com/thesofproject/linux/blob/topic/sof-dev/include/sound/sof/dai-amd.h#L20